components/video-card/video-card.js

```diff
@@ -16,6 +17,16 @@ function videoCardDirective() {
 		scope.categories = scope.video.categories;
 		scope.productType = scope.video.productType;
 		scope.verified = scope.video.verified;
+		scope.owner = scope.video.owner;
+		scope.showPublishedIcon = showPublishedIcon(scope.owner, scope.published);
+	}
+
+	function showPublishedIcon(owner, published) {
+		if (session.role === 'editor' || (owner === session.id && $state.current.name === 'userGallery')) {
+			return (published === true || published === 'true');
+		} else {
+			return false;
```

Comment:
we can improve this code that way: 
```
if (session.role === 'editor' || (owner === session.id && $state.current.name === 'userGallery')) {
   return (published === true || published === 'true');
}
return false;
```

Reply:
+1, in video-detail-controller we have the same mistake.